GsaGHTests/Helpers/Assemble/AssembleModelLists.cs

```diff
@@ -0,0 +1,56 @@
+﻿using GsaGH.Components;
+using GsaGH.Parameters;
+
+using Xunit;
+
+namespace GsaGHTests.Helpers.Assemble {
+  [Collection("GrasshopperFixture collection")]
+  public class AssembleModelLists {
+    private static readonly int DefaultId = 1;
+    private static readonly string DefaultName = "create list test";
+    private static readonly string DefaultDefinition = "1 2 3";
+
```

Comment:
@DominikaLos @psarras Problem is in assembling list and not on component. So, my personal opinion would be to test assembly.

[Fact]
public void ListWithDefinitionAreAssembledCorrectly() {
  var model = new GsaModel();
  var gsaLists = new GsaList(DefaultName, DefaultDefinition, GsaAPI.EntityType.Node) {
    Id = DefaultId
  };
  var assembly = new ModelAssembly(model, new List<GsaList> { gsaLists }, null, null, null, null, null,
  OasysUnits.Units.LengthUnit.Meter, Length.Zero, false, null);
  Assert.Single(assembly.GetModel().Lists());
  Assert.Equal(DefaultDefinition, assembly.GetModel().Lists().First().Value.Definition);
}

[Fact]
public void ListWithNodesAreAssembledCorrectly() {
  var model = new GsaModel();
  var gsaLists = new GsaList(DefaultName, "", GsaAPI.EntityType.Node) {
    Id = DefaultId
  };
   var listOfNodes = new List<GsaNodeGoo> {
     new GsaNodeGoo(new GsaNode(new Rhino.Geometry.Point3d(0, 0, 0))),
     new GsaNodeGoo(new GsaNode(new Rhino.Geometry.Point3d(0, 0, 1))),
     new GsaNodeGoo(new GsaNode(new Rhino.Geometry.Point3d(0, 0, 2)))
   };
  gsaLists.SetListGooObjects(listOfNodes.Cast<object>().ToList());
  var assembly = new ModelAssembly(model, new List<GsaList> { gsaLists }, null, null, null, null, null,
  OasysUnits.Units.LengthUnit.Meter, Length.Zero, false, null);
  Assert.Single(assembly.GetModel().Lists());
  Assert.Equal(DefaultDefinition, assembly.GetModel().Lists().First().Value.Definition);
}